### DISCUSSION TITLE
load_type_definitions not working

### DISCUSSION THREAD

**Discussion by acuna-luis** [FreeOpcUa/python-opcua#1506] [General]
Hello, I am working with a S7-1200 but I can't read a structure properly. My output before and after load_type_definitions

**BEFORE**
node object ns=4;i=5
node object value ExtensionObject(TypeId:FourByteNodeId(ns=4;i=4), Encoding:1, 2176 bytes)
params obj {'server': <opcua.client.ua_client.UaClient object at 0x7f4fa8452f40>, 'nodeid': NumericNodeId(ns=4;i=5), 'basenodeid': None}

**AFTER**  self.client.load_type_definitions()
node object ns=4;i=5
node object value ExtensionObject(TypeId:FourByteNodeId(ns=4;i=4), Encoding:1, 2176 bytes)
params obj {'server': <opcua.client.ua_client.UaClient object at 0x7f4fa8452f40>, 'nodeid': NumericNodeId(ns=4;i=5), 'basenodeid': None}
'_freeze': True}

And when accessing an existing atribute I got error.
**ERROR**
OPCUA Exception: <class 'AttributeError'>
Traceback (most recent call last):
  File "/home/lacuna/dsa/internal/hw-dashboard/flow-control-operator/modules/devices.py", line 549, in update_DB
    print("hola", hola.disponible)
AttributeError: 'ExtensionObject' object has no attribute 'disponible'

The code is:
```python              self.db.obj = self.client.get_node('ns=4;i=5')
                print("BEFORE")
                print("node object", self.db.obj)
                print("node object value", self.db.obj.get_value())
                print("params obj",get_object_params(self.db.obj))
                print("params val",get_object_params(self.db.obj.get_value()))
                print("AFTER")
                self.client.load_type_definitions()
                print("node object", self.db.obj)
                print("node object value", self.db.obj.get_value())
                print("params obj",get_object_params(self.db.obj))
                print("params val",get_object_params(self.db.obj.get_value()))
                var1 = self.db.obj.get_value()
                print("var1", ``var1.disponible)```

Please, your help if you have any idea. Thanks

**Comment by schroeder-** [comment]
This library is not supported anymore try asyncua
